distribution/kernel/carbon-home/dbscripts/db2.sql

```diff
@@ -607,6 +607,7 @@ CREATE TABLE UM_PERMISSION(
     UM_ACTION VARCHAR(255) NOT NULL,
     UM_TENANT_ID DECIMAL(31,0) DEFAULT 0 NOT NULL,
     UM_MODULE_ID INTEGER DEFAULT 0,
+    UNIQUE(RESOURCE_ID, ACTION),
```

Comment:
This should be UM_RESOURCE_ID and UM_ACTION.          

BTW these two will not be unique.

should be corrected as follows
 UNIQUE(UM_RESOURCE_ID,UM_ ACTION, UM_TENANT_ID),